tests/hp/hp_constraints_q.output

```diff
@@ -29607,7 +29607,7 @@ DEAL::n_dofs=2970
     2923 613:  -0.07407407
     2923 720:  0.2500000
     2923 1105:  -0.01851852
-    2923 1106:  -2.775558e-17
+    2923 1106:  -1.387779e-17
```

Comment:
It really seems that there are just a few dofs that switched indices with each other. I found these constraints at other stages in this output. I don't think it's necessary to verify them by e.g. comparing histograms on the constrained values, since as you said, just some orientation changed.